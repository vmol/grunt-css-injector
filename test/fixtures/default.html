<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<title>Document</title>
	<style type="text/css"></style>
</head>
<body>
	<h1>Hello World</h1>
	<p><a href="#">hello world</a></p>
</body>
</html>