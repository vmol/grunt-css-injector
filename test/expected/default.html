<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<title>Document</title>
	<style type="text/css">body{color:#fff;}a{color:#fea;}</style>
</head>
<body>
	<h1>Hello World</h1>
	<p><a href="#">hello world</a></p>
</body>
</html>